<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TalkRoom</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>30</y>
     <width>201</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="sendText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>400</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>431</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findipButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自动寻找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelconnect">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>未连接</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
